### PR TITLE
[TEST] Setup coveralls

### PR DESCRIPTION
**Pull request by YukieK**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -11,8 +11,15 @@ envlist =
 skip_missing_interpreters = true
 
 [testenv]
-deps = pytest
+passenv = CIRCLECI CIRCLE_* CI_PULL_REQUEST COVERALLS_PARALLEL COVERALLS_REPO_TOKEN COVERALLS_SERVICE_NAME
```
Comment (mitsutaka) [inline tox.ini]: export `COVERALLS_PARALLEL`  when `coverage` executes. You don't need to setup on CircleCI.

File: tox.ini [hunk 2]
```diff
@@ -11,8 +11,15 @@ envlist =
...
+deps =
+    pytest
+    coveralls
 whitelist_externals = make
 commands =
     make test_all
     pip3 install geekhours
+    coverage run --source=geekhours setup.py test
+    coverage report -m --omit=.tox/*
+    coveralls --verbose
```
Comment (mitsutaka) [inline tox.ini]: Let's omit verbose log. We confirmed coveralls works well.

File: .circleci/config.yml [hunk 3]
```diff
@@ -49,13 +49,9 @@ jobs:
                 command: |
                   echo 'export PATH=$HOME/.local/bin:$PATH' >> $BASH_ENV
                   source $BASH_ENV
-            - run:
-                name: Unit test on python 3.5
-                command: make test_all
             - run:
                 name: Install geekhours package on python 3.5
```
[the remark this answers is not in the document]
Comment (YukieK) [reply .circleci/config.yml]: Fixed.

File: .circleci/config.yml [hunk 4]
```diff
@@ -65,20 +61,21 @@ jobs:
                 command: |
                   echo 'export PATH=$HOME/.local/bin:$PATH' >> $BASH_ENV
                   source $BASH_ENV
-            - run:
-                name: Unit test on python 3.6
-                command: make test_all
             - run:
                 name: Install geekhours package on python 3.6
```
[the remark this answers is not in the document]
Comment (YukieK) [reply .circleci/config.yml]: Fixed.

File: README.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 [![CircleCI](https://img.shields.io/circleci/project/github/YukieK/GeekHours.svg?style=flat-square)](https://circleci.com/gh/YukieK/GeekHours)
 [![PyPI version](https://img.shields.io/pypi/v/GeekHours.svg?color=%2351A5DC&style=flat-square)](https://pypi.org/project/geekhours/)
 [![GitHub license](https://img.shields.io/github/license/YukieK/geekhours.svg?color=%23e84566&style=flat-square)](https://github.com/YukieK/GeekHours/blob/master/LICENSE)
+[![Coverage Status](https://coveralls.io/repos/github/YukieK/GeekHours/badge.svg?branch=setup_coveralls)](https://coveralls.io/github/YukieK/GeekHours?branch=setup_coveralls)
```
[the remark this answers is not in the document]
Comment (YukieK) [reply README.md]: Fixed.

File: tox.ini (hunk 1 above)
Comment (YukieK) [reply tox.ini]: Only COVERALLS_REPO_TOKEN was needed.

File: tox.ini (hunk 2 above)
Comment (YukieK) [reply tox.ini]: Fixed.

File: .circleci/config.yml [hunk 6]
```diff
@@ -65,20 +61,21 @@ jobs:
...
             - checkout
             - run:
-                name: Unit test
-                command: make test_all
+                name: Make Composer executables available globally
+                command: |
+                  echo 'export PATH=$HOME/.local/bin:$PATH' >> $BASH_ENV
+                  source $BASH_ENV
+            - run:
+                name: Install geekhours package on python 3.7
```
[the remark this answers is not in the document]
Comment (YukieK) [reply .circleci/config.yml]: Fixed.

**Pull request merged**